core/src/main/java/org/apache/calcite/util/TimeString.java

```diff
@@ -76,35 +76,54 @@ private static String hms(int h, int m, int s) {
     return b.toString();
   }
 
-  /** Sets the fraction field of a {@code TimeString} to a given number
-   * of milliseconds. Nukes the value set via {@link #withNanos}.
+  /**
```

Comment:
keep formatting (no newline after `/**`)

There are several arbitrary changes in this PR, each of which take the documentation further away from the parallel documentation in `TimestampString`.